src/doc/functional/_cms.rst

```diff
@@ -57,7 +60,21 @@ Services
 
 ----------
 
-.. py:function:: readCredential(credentialID, filter, transactionID)
+.. py:function:: findCredentials(expressions, transactionID)
```

Comment:
Will this be usable for 3rd Party Services? Are you going to add it to the table here:
https://osia.readthedocs.io/en/latest/02%20-%20functional.html#components-vs-interfaces-mapping